### ISSUE TITLE
Accessibility - Font sizes screen Import kit too small

### ISSUE THREAD

**Issue by Bovelett** [elementor/elementor#23860]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The font sizes of the Kit import screen are so small that it's hard to read. Main paragraph .eps-text is set to 0.875rem. Text in notice is .75 rem and .eps-text-sm is set to .8125rem. 

Although WCAG guidelines do not dictate a minimum font-size, most leading accessibility specialists recommend a minimum of 16px. Besides, I may not be the only user with bad eyesight. 

![Screenshot 2023-09-20 at 10 18 33](https://github.com/elementor/elementor/assets/852140/6f2aa0dc-8541-4619-a7bb-7de5d202c542)

![Screenshot 2023-09-20 at 10 39 09](https://github.com/elementor/elementor/assets/852140/ea79911a-b8c7-43e1-b7d0-d9da0e45063a)

❤️ I would ask you to please let common sense prevail and do not go under the equivalent of 16px. This screen leaves enough space to do so, don't you think?

### Steps to reproduce

Import a site kit in a site with an existing header and footer template.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Ubuntu 22.04 v10.6.12
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 200M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 200 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Berlin
	Admin Email: xxxxxxxx
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.16.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.1
		Author: Elementor.com

== Elements Usage ==
	
	archive : 2
		button : 1
		heading : 3
		posts : 1
		text-editor : 1
	footer : 2
		form : 1
		heading : 5
		icon-list : 1
		nav-menu : 2
		social-icons : 1
		theme-site-logo : 1
	header : 2
		nav-menu : 2
		theme-site-logo : 2
	page : 2
		button : 3
		heading : 23
		image : 1
		spacer : 5
	popup : 1
		form : 1
		heading : 1
		text-editor : 1
	section : 1
		form : 1
	wp-page : 4
		button : 6
		call-to-action : 12
		divider : 13
		form : 1
		google_maps : 1
		heading : 43
		image : 1
		image-carousel : 1
		posts : 1
		spacer : 8
		testimonial-carousel : 1
		text-editor : 32
	single-post : 1
		button : 1
		heading : 1
		post-info : 1
		post-navigation : 1
		posts : 1
		share-buttons : 1
		theme-post-content : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	error-404 : 1
		button : 1
		heading : 2

== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default

== Log ==
	
Log: showing 20 of 412023-09-20 09:26:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.10.3',
  'to' => '3.16.1',
)]
2023-09-20 09:26:45 [info] elementor::elementor_updater Started 
2023-09-20 09:26:45 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-20 09:26:45 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-20 09:26:45 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2023-09-20 09:26:45 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2023-09-20 09:26:45 [info] Elementor/Upgrades - _v_3_16_0_container_updates Start  
2023-09-20 09:26:45 [info] Elementor/Upgrades - _v_3_16_0_container_updates Finished 
2023-09-20 09:26:45 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-20 09:26:45 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-20 09:26:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.7',
  'to' => '3.16.3',
)]
2023-09-20 10:11:59 [info] Uploading Kit:  [array (
  'kit_id' => '6243029b1475e1001dc79b23',
  'referrer' => 'kit-library',
)]
2023-09-20 10:36:29 [info] Uploading Kit:  [array (
  'kit_id' => '6243029b1475e1001dc79b23',
  'referrer' => 'kit-library',
)]
2023-09-20 10:36:36 [info] Selected import runners: site-settings, plugins, templates, taxonomies, elementor-content, wp-content, elements-default-values 
2023-09-20 10:36:37 [info] Import runner completed: site-settings ✓ 
2023-09-20 10:36:37 [info] Import runner completed: plugins ✓ 
2023-09-20 10:36:38 [info] Import runner completed: templates ✓ 
2023-09-20 10:36:39 [info] Import runner completed: taxonomies ✓ 
2023-09-20 10:36:49 [info] Import runner completed: elementor-content ✓ 
2023-09-20 10:36:56 [info] Import runner completed: wp-content ✓ 

PHP: showing 2 of 2PHP: 2023-09-20 10:00:44 [notice X 20][/www/htdocs/w01aa8bf/xxxxx.xx/wp-content/plugins/elementor/includes/libraries/bfi-thumb/bfi-thumb.php::750] A non well formed numeric value encountered [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-20 10:36:38 [notice X 1][/www/htdocs/w01aa8bf/xxxxx.xx/wp-content/plugins/elementor-pro/core/app/modules/import-export/runners/import/templates.php::103] Undefined index: conditions [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

**Comment by nicholaszein** [comment]
Hey @Bovelett ❤️! 

Thank you for submitting your issue! 🙏 

I've forwarded this to the team responsible for the UI. 

I can't say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 🙂 

Kind regards
